cmd/create/cluster/cmd.go

```diff
@@ -1015,8 +1015,8 @@ func run(cmd *cobra.Command, _ []string) {
 	httpTokens := args.ec2MetadataHttpTokens
 	if interactive.Enabled() && !isHostedCP {
 		httpTokens, err = interactive.GetString(interactive.Input{
-			Question: fmt.Sprintf("Configure the use of IMDSv2 for ec2 instances %s/%s",
-				v1.Ec2MetadataHttpTokensOptional, v1.Ec2MetadataHttpTokensRequired),
+			Question: "Should cluster nodes use both v1 and v2 endpoints or just v2 endpoint " +
+				"of EC2 Instance Metadata Service (IMDS)",
```

Comment:
I dont think this is what is asked in the JIRA. 

I think we need to update the help text and not the question.